mozetl/basic/transform.py

```diff
@@ -40,7 +40,7 @@ def _build_cell(ping, column_config):
     if func is not None:
         try:
             return func(raw_value)
-        except:  # TODO: what exception could happen here, fill in?
+        except Exception:
```

Comment:
I actually made this change in #181, we should probably merge that instead.

Reply:
SGTM, I'll remove this change.